EditView.vala

```diff
@@ -71,7 +71,12 @@ class EditView: Gtk.DrawingArea, Gtk.Scrollable {
 		blink_time = settings.get_int("cursor-blink-time") / 2;
 		can_focus = true;
 		set_has_window(true);
-		add_events(Gdk.EventMask.BUTTON_PRESS_MASK|Gdk.EventMask.BUTTON_RELEASE_MASK|Gdk.EventMask.BUTTON_MOTION_MASK|Gdk.EventMask.SCROLL_MASK|Gdk.EventMask.SMOOTH_SCROLL_MASK);
+		add_events(Gdk.EventMask.BUTTON_PRESS_MASK |
+			Gdk.EventMask.BUTTON_RELEASE_MASK |
+			Gdk.EventMask.BUTTON_MOTION_MASK |
+			Gdk.EventMask.SCROLL_MASK |
+			Gdk.EventMask.SMOOTH_SCROLL_MASK);
```

Comment:
Could you please leave these in one line? IMHO line breaking is the job of the editor, not the programmer. I'm fine with adding spaces around the operator, though.